plugins/offboard/offboard.h

```diff
@@ -81,6 +92,42 @@ class Offboard : public PluginBase {
                           above). */
     };
 
+    /**
+     * @brief Type for Velocity commands in NE (North East) and Altitude in N (Down) coordinate and
+     * yaw.
+     */
+    struct VelocityAltitudeNEDYaw {
+        float north_m_s; /**< @brief Velocity North in meters/second. */
+        float east_m_s; /**< @brief Velocity East in meters/second. */
+        float down_m; /**< @brief Altitude Down in meters. */
+        float yaw_deg; /**< @brief Yaw in degrees (0 North, positive is clock-wise looking from
+                          above). */
+    };
+
+    /**
+     * @brief Type for Position commands in NE (North East) and Velocity in N (Down) coordinate and
+     * yaw.
+     */
+    struct PositionClimbRateNEDYaw {
+        float north_m; /**< @brief Position North in meters. */
+        float east_m; /**< @brief Position East in meters. */
+        float down_m_s; /**< @brief Velocity Down in meters/second. */
+        float yaw_deg; /**< @brief Yaw in degrees (0 North, positive is clock-wise looking from
+                          above). */
+    };
+
+    /**
+     * @brief Type for Position commands in NE (North East) and Velocity in N (Down) coordinate and
+     * yawspeed.
```

Comment:
Is this an actual use case?

Reply:
it was mainly added for @darioxz. He has a version where he uses his own "yaw-controller" that spits out yawspeed setpoints.